.github/workflows/maven.yml

```diff
@@ -49,10 +49,9 @@ jobs:
           distribution: adopt
           java-version: ${{ matrix.java }}
       - name: Build with Maven on Java ${{ matrix.java }}
-        if: matrix.java != '11'
         run: mvn -B -V -DskipAssembly test --no-transfer-progress
```

Comment:
Shouldn't this be `verify` to ensure we run the integration tests for every JDK?

Reply:
Right, will change that

Reply:
This can lead to failed tests as quite often builds of different PRs are run on the same build agent and Jetty can not bind to port 8090 which already used. I will keep `test` phase for the JDK 8 & 11.